test/suites/local/microformats-v2/rel-urls.json

```diff
@@ -6,7 +6,7 @@
   },
   "rel-urls": {
     "http://example.com": {
-      "rels": ["me", "home", "example"],
+      "rels": ["example", "home", "me"],
```

Comment:
I think I need to look more into why the order changed. Also is causing the failing test.

Reply:
@aciccarello I've already organised a fix for you here: https://github.com/aciccarello/microformats-parser/pull/1 - because we're having to reference the git repository for the testing suite dependency it needs forcing to update 😄